source/src/scripts/FocusMode.js

```diff
@@ -16,7 +16,9 @@ function toggleState() {
     button.classList.toggle('state');
     if (localStorage.getItem('state') === 'default') {
         localStorage.setItem('state', 'focus');
+        document.getElementById('main').style.display = 'block';
```

Comment:
Instead of document.getElementById('main') consider removing the id from main and using document.querySelector('main'). Redundant otherwise.